 <launch>
    <node name="position_track_setpoint" pkg="hku_m100_landing" type="position_track_setpoint" output="screen" >
    <param name="apriltags_36h11_sub" value="/apriltags/36h11/detections" />
    <param name="apriltags_16h5_sub" value="/apriltags/16h5/detections" />
    
    </node>
 </launch>